forc-pkg/src/pkg.rs

```diff
@@ -827,10 +828,25 @@ pub fn graph_to_path_map(
                             bail!("missing path info for dependency: {}", &dep_name);
                         }
                     })?;
-                let rel_dep_path = detailed
-                    .path
-                    .as_ref()
-                    .ok_or_else(|| anyhow!("missing path info for dependency: {}", dep.name))?;
+                // Check if there is a patch for this dep
+                let patch = parent_manifest
+                    .patches()
+                    .find(|patches| patches.1.contains_key(&dep_name))
+                    .and_then(|patches| patches.1.get(&dep_name));
```

Comment:
`find` and `and_then` used in succession can often be replaced with `find_map` - not important, just a tip!

Reply:
Oh, thanks :) that looks better